src/testing/models.ts

```diff
@@ -8,6 +8,7 @@ export interface Threshold {
 export interface Evaluation {
   score: number;
   threshold?: Threshold;
+  metadata?: Record<string, string>;
```

Comment:
Shouldn't the value be any

Reply:
Was thinking about DB - but the ingestion worker flattens. Switching to unknown